xbmc/Autorun.cpp

```diff
@@ -77,7 +77,7 @@ void CAutorun::ExecuteAutorun(const std::string& path, bool bypassSettings, bool
 
   CCdInfo* pInfo = CServiceBroker::GetMediaManager().GetCdInfo(path);
 
-  if ( pInfo == NULL )
+  if ( !pInfo )
```

Comment:
Additional spaces after parenthesis. Clang-format should find such stuff isn't it?

Reply:
Yep I will apply jenkins diff separated in each commit for that. Also some code that occupies two lines can now be fitted into a single line.

Reply:
You can also do post-commit hooks. A team member pointed me to https://github.com/barisione/clang-format-hooks.